src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,153 @@
+package duke;
+
+import duke.exceptions.DukeInvalidArgumentException;
+import duke.tasks.*;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+
+    private static ArrayList<Task> taskList;
+    private static Scanner getUserInput;
```

Comment:
getUserInput sounds like a method name. Perhaps consider changing it to 
```suggestion
    private static Scanner sc;
```